### ISSUE TITLE
Readme.md for chai-exec plugin is out of date

### ISSUE THREAD

**Issue by eduard-malakhov** [chaijs/chaijs.github.io#191]
Readme.md for chai-exec (plugins/chai_exec.md) hasn't been updated since 10 Jan 2019. Meanwhile, there have been changes in the original readme (https://github.com/chaijs/chaijs.github.io/blob/master/plugins/chai_exec.md) including badge links which are now broken at https://www.chaijs.com/plugins/chai-exec/

I am now aware of the mechanism for updating these readme's and who's responsibility it is, but I thought it would be nice to bring the docs in sync with the source.

@JamesMessinger, you might want to take a look at this.

**Comment by keithamus** [comment]
Hm, we had Travis running in a cron job, but I guess the service is no longer running properly. Migrating to GitHub Actions might be worthwhile now.

**Comment by eduard-malakhov** [comment]
@keithamus,  the last commit from @chaijs-bot was on Jun 10, 2019 (fac9905d0026cff89f411c236a4ad0ca36942f32)... Have you planned migrating to GitHub actions yet?

**Comment by keithamus** [comment]
I've been very busy with my personal life recently, too busy to spend any significant amount of time with Chai. I haven't planned migrating this particular build to GitHub Actions, but we've been using it more and more throughout other Chai projects, so I think it would work well here.

**Comment by eduard-malakhov** [comment]
@keithamus, can you point me to some repos where you've moved to GitHub actions already? I am now starting to migrate some of my projects and I think I could draft actions for this repo too, but I'd like to see some examples to ensure consistency across different Chai-related projects.

**Comment by keithamus** [comment]
@eduard-malakhov `loupe` runs on GitHub Actions now: https://github.com/chaijs/loupe/tree/master/.github/workflows, as does `chai-http`: https://github.com/chaijs/chai-http/tree/master/.github/workflows

**Comment by keithamus** [comment]
@eduard-malakhov it's possible to run GH Actions on commit to master and also on cron job - which would be useful for this repo

**Comment by eduard-malakhov** [comment]
@keithamus, thank you, I will take a look. What was the schedule for Travis job?

**Comment by keithamus** [comment]
It ran daily I think! Perhaps at midnight UTC? I don’t think it’s import so much when, as long as it runs regularly.

**Comment by eduard-malakhov** [comment]
There's only a limited number of free compute hours for GA per account, so daily runs might generate extra costs. There have been 34 commits from @chaijs-bot during 6 the last six month when the builds were running, which gives us approx. one commit every 4 days. We can use this for reference.

**Comment by JamesMessinger** [comment]
Hi guys.  I just now saw this issue.  Am I correct in understanding that this isn't a problem with chai-exec, but rather with an automated job that builds the Chai.js website?

**Comment by keithamus** [comment]
@JamesMessinger yes that's correct. The Chai.js website hasn't been updated for 6 months.

**Comment by keithamus** [comment]
> There's only a limited number of free compute hours for GA per account, so daily runs might generate extra costs.

GitHub Actions is free and unlimited for public (aka FOSS) repos (within the usage policy https://help.github.com/en/actions/getting-started-with-github-actions/about-github-actions#usage-limits). Private repositories cost in minutes, but this repo is not private. Rest assured we can run the job daily @eduard-malakhov 😄 😉
